src/components/enterprise-page/data/hooks.js

```diff
@@ -25,13 +26,18 @@ const defaultBrandingConfig = {
  */
 export function useEnterpriseCustomerConfig(enterpriseSlug) {
   const [enterpriseConfig, setEnterpriseConfig] = useState();
+  let display404Page = false;
 
   useEffect(() => {
     fetchEnterpriseCustomerConfig(enterpriseSlug)
       .then((response) => {
         const { results } = camelCaseObject(response.data);
         const config = results.pop();
-        if (config?.enableLearnerPortal) {
+        if (!config) {
```

Comment:
The conditional `config?.enableLearnerPortal` is already handling when `config` is undefined; I don't think we need to separate these conditionals out when both cases are calling `setEnterpriseConfig(null)` already.